### PR TITLE
ISPN-14319 Use InetSocketAddress.getHostString() instead of .getHostName()

### PR DESCRIPTION
**Pull request by tristantarrant**
https://issues.redhat.com/browse/ISPN-14319

### CODE REVIEW

**Review by pruivo — APPROVED** [review]

File: server/core/src/main/java/org/infinispan/server/core/transport/NettyTransport.java [hunk 1]
```diff
@@ -231,7 +231,7 @@ public long getTotalBytesRead() {
    )
    @Override
    public String getHostName() {
-      return address.getHostName();
+      return address.getHostString();
```
[the remark this answers is not in the document]
Comment (tristantarrant) [reply server/core/src/main/java/org/infinispan/server/core/transport/NettyTransport.java]: ok

**Comment by tristantarrant** [comment]
I've fixed the Kerberos stuff. Let's see what CI says

**Comment by tristantarrant** [comment]
This is now good to go. I'll create a backport

**Comment by pruivo** [comment]
merged! thanks @tristantarrant !

**Pull request merged**